Formula/grin-wallet.rb

```diff
@@ -0,0 +1,17 @@
+class GrinWallet < Formula
+  desc "Official wallet for the cryptocurrency Grin"
+  homepage "https://grin.mw"
+  url "https://github.com/mimblewimble/grin-wallet/archive/v2.1.0.tar.gz"
+  sha256 "649664af929fc2afd42e52597ad675d3e1856c95e0cda60ef8d4c478abe5c01c"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", "--root", prefix, "--path", "."
```

Comment:
Sorry one more thing: Do you have a cargo.lock file? If so can you please add `--lock` to the install? If grinwallet doesn't have one, what would it take to convince upstream to add one?

Reply:
We do. Adding the `--locked` flag.

Reply:
Thanks! Glad you didn't add `--lock` instead of `--locked` too!